core/project/index.js

```diff
@@ -66,6 +66,7 @@ class Project {
             // No need to distinguish between missing and empty,
```

Comment:
Obviously I can't speak to the intent behind this comment originally, but your change does make it seem a little nonsensical, given you are adding an `isEmpty` right below a comment saying "no need to distinguish between missing and empty".

We could probably just amend the `this.configNotFound` assignment to check for `Object.keys(loaded.config ?? {}).length === 0`?

Reply:
You are right that is a little bit weird. Just changed that and used your code suggestion.